litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-LIMIT-RESOURCE/run_litmus_test.yml

```diff
@@ -67,7 +67,7 @@ spec:
 
           ## Enter exclude device filter  
           - name: EXCLUDE_DEVICE_FILTER
-            value: ''
+            value: '/dev/sdb'
```

Comment:
@amitbhatt818 will it be better to add this value in the script where this job is being run, instead of hard coding it here?

Reply:
@harshshekhar15 we are going to use same job file in all the platforms thatswhy I added this here .

Reply:
Ok. Let's add some comment specifying why we are excluding the following device.

Reply:
Ok sure

Reply:
are we passing this value while openebs deployment

Reply:
Yes @gprasath